config/initializers/rails_config.rb

```diff
@@ -1,3 +1,4 @@
 Config.setup do |config|
   config.const_name = "Settings"
+  config.use_env = true
```

Comment:
Do you also need to set:
```
config.env_separator = '__'
config.env_converter = :downcase
```

Reply:
looks like `:downcase` is the default, but the default separator is `.`, so I do need to set that. good catch

Reply:
updated